src/Resources/ShipmentStatus.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+namespace MyParcelCom\ApiSdk\Resources;
+
+use MyParcelCom\ApiSdk\Resources\Interfaces\ResourceInterface;
+use MyParcelCom\ApiSdk\Resources\Interfaces\ShipmentInterface;
+use MyParcelCom\ApiSdk\Resources\Interfaces\ShipmentStatusInterface;
+use MyParcelCom\ApiSdk\Resources\Interfaces\StatusInterface;
+use MyParcelCom\ApiSdk\Resources\Traits\JsonSerializable;
+
+class ShipmentStatus implements ShipmentStatusInterface
+{
+    use JsonSerializable;
+
+    const ATTRIBUTE_CODE = 'carrier_status_code';
```

Comment:
Change to `ATTRIBUTE_CARRIER_CODE`?
Might make it more clear when other people use these constants.